docs/source/tutorials_source/package/tutorial_simsiam_esa.py

```diff
@@ -0,0 +1,447 @@
+"""
+.. _lightly-simsiam-tutorial-4:
+
+Tutorial 4: Train SimSiam on Satellite Images
+==============================================
+
+In this tutorial we will train a SimSiam model in old-school PyTorch style on a
+set of satellite images of Italy. We will showcase how the generated embeddings
+can be used for exploration and better understanding of the raw data.
+
+You can read up on the model in the paper
+`Exploring Simple Siamese Representation Learning <https://arxiv.org/abs/2011.10566>`_.
+
+We will be using a dataset of satellite images from ESAs Sentinel-2 satellite over Italy.
+The original images have been cropped into smaller tiles due to their immense size and
+the dataset has been balanced based on a simple clustering of the mean RGB color values
+to prevent a surplus of images of the sea.
+
+In this tutorial you will learn:
+
+- How to work with the SimSiam model
+
+- How to do self-supervised learning in old-school PyTorch style
+
+- How to check whether your embeddings have collapsed
+
+"""
+
+# %%
+# Imports
+# -------
+#
+# Import the Python frameworks we need for this tutorial.
+
+
+import math
+import torch
+import torch.nn as nn
+import torchvision
+import numpy as np
+import lightly
+
+
+# %%
+# Configuration
+# -------------
+# 
+# We set some configuration parameters for our experiment. 
+#
+# The default configuration with a batch size and input resolution of 128
+# requires 16GB of GPU memory.
+
+num_workers = 8
+batch_size = 128
+seed = 1
+epochs = 30
+input_size = 256
+
+# dimension of the embeddings
+num_ftrs = 512
+# dimension of the output of the prediction and projection heads
+out_dim = proj_hidden_dim = 512
+# the prediction head uses a bottleneck architecture
+pred_hidden_dim = 128
+# use 2 layers in the projection mlp
+num_mlp_layers = 2
+
+
+# %%
+# Let's set the seed for our experiments and the path to our data
+
+
+# seed torch and numpy
+torch.manual_seed(0)
+np.random.seed(0)
+
+# set the path to the dataset
+path_to_data = '/datasets/sentinel-2-italy-v1/'
+
+
+# %%
+# Setup data augmentations and loaders
+# ------------------------------------
+# Since we're working on satellite images, it makes sence to use horizonal and
+# vertical flips as well as random rotation transformations. We apply weak color 
+# jitter to learn an invariance of the model with respect to slight changes in
+# the color of the water.
+#
+
+# the collate function defines the transformations for self-supervised learning
```

Comment:
The term 'data augmentation' is never used here, even though it is the most common word for such operations (not 'transformations' or even 'collate functions'. This confused me.